src/tests/instrument_drivers/virtualAwg/test_virtual_awg.py

```diff
@@ -1,28 +1,41 @@
 import unittest
 from qcodes import Instrument
+
+from qtt.instrument_drivers.virtualAwg.settings import SettingsInstrument
 from qtt.instrument_drivers.virtualAwg.virtual_awg import VirtualAwg
 
 
+class QuantumDeviceSettings(Instrument):
```

Comment:
This class is nowhere used.

Reply:
Yes, it's replaced by `SettingsInstrument` now. Reworked and removed.